_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Handbook Of Manners: How To Behave In A Code Review"
+description: Some thoughts on code reviews and the way they usually are and they should be performed.
+modified:
+categories: blog
+author: henning_gross
+excerpt: Code reviews are a common practise for knowledge distribution, learning and increasing quality - but do we perform them in a way that actually creates value?
```

Comment:
To avoid confusion use: 
> practice

practice/practise is the same for US English (which you have generally used), but it's noun/verb in British English; so it looked a little odd to me.